static/app/bootstrap/initializeSdk.tsx

```diff
@@ -76,6 +76,11 @@ export function initializeSdk(config: Config, {routes}: {routes?: Function} = {}
      * as well as `whitelistUrls`
      */
     dsn: SPA_DSN || sentryConfig?.dsn,
+    /**
+     * Frontend can be built with a `SENTRY_RELEASE_VERSION` environment variable for release string, useful if frontend is
+     * deployed separately from backend.
+     */
+    release: SENTRY_RELEASE_VERSION || sentryConfig?.release,
```

Comment:
`||` or `??`

Reply:
Hmm, `??` if we think `SENTRY_RELEASE_VERSION=0` is an intended version

Reply:
```suggestion
    release: SENTRY_RELEASE_VERSION ?? sentryConfig?.release,
```